lnwire/message.go

```diff
@@ -316,3 +317,33 @@ func ReadMessage(r io.Reader, pver uint32, btcnet wire.BitcoinNet) (int, Message
 
 	return totalBytes, msg, payload, nil
 }
+
+// MessageToStringClosure is used to produce less spammy log messages in trace
+// mode by setting the 'Curve" parameter to nil. Doing this avoids printing out
+// each of the field elements in the curve parameters for secp256k1.
+func MessageToStringClosure(msg Message) logClosure {
```

Comment:
I don't think this belongs in the wire package. The prior method was meant to be isolated to the logging within the peer struct.